src/components/Rout.tsx

```diff
@@ -0,0 +1,29 @@
+import { Routes, Route, Navigate } from 'react-router-dom';
+import { TabsPage } from './TabsPage';
+
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const Rout = () => {
```

Comment:
The function name 'Rout' is likely a misspelling. Consider renaming it to 'Route'.